2_Operations/apps.md

```diff
@@ -2,53 +2,60 @@
+- **Sound & appearance**: check `Mute all sounds from Slack`, audio pings are extremely distracting. Uncheck `Bounce Slack's icon when receiving a notification`, another really annoying distraction. Unchecking `Include a preview of the message in each notification` will help you keep focus by notifying you of the message but not what it's about (where your brain automatically starts thinking about it). If you always need Slack empty of the red icon and click through everything to "clean it", uncheck `Show a badge on Mac on Slack's icon to indicate new activity`. 
 
 We encourage you to **quit Slack** when you need time for undistracted work. Make sure that other Niteans that depend on your feedback know that you're away. But unless there's a fire, a few hours of uninterrupted work will only benefit everyone. 
 
 You can also use the **bell icon** (Notifications) to disable all notifications for a specific time period. However, this still shows the number of received messages in the Slack app icon (which is a distraction in itself).
 
 
+#### Slack is Unsecure Messaging
+
+Although we use Slack for our day-to-day communication, it is not a fully secure messaging system since the messages are stored and accessible on their servers. Therefore do not transfer passwords, keys or confidential information using Slack. Instead, use Signal, which is secured with end-to-end encryption.
+
+#### Other Slack tips
+
+- Don't use naked pings, where you type "hey" and send, and only then start writing your message. A greeting is welcome, but keep it inside the main message.
+- If you hover over any message, there is a ribbon you can use to save a message. You can use this if it's something you see referencing or needing it in the future.
+- Use reminders to "snooze" a message. This is really beneficial for Friday messages where you can postpone it to first thing Monday.
+
 ## Signal
 
-[Signal](https://signal.org/) is a secure end-to-end encrypted (E2EE) messaging application. We use it for confidential messages that should not be sent via email or Slack. To start using Signal you will require an Android phone or iPhone and a working phone number.
+[Signal](https://signal.org/) is a secure end-to-end encrypted (E2EE) messaging application. We use it for confidential messages that should not be sent via email or Slack. To start using Signal, you will require an Android phone or iPhone and a working phone number.
 
 #### Usage
 
-Signal is to be used primarily for sending confidential information that must never be shared over Slack, and timely instant messaging at conferences and IRLs. All other instant messaging should remain on Slack.
+Signal is to be used primarily for sending confidential information that must never be shared over Slack. All other instant messaging should remain on Slack.
```

Comment:
I'd keep this. I don't have Slack on my phone, but I do have Signal, since Signal is used for urgent stuff.